common/src/main/java/io/novafoundation/nova/common/utils/InformationSize.kt

```diff
@@ -1,7 +1,7 @@
 package io.novafoundation.nova.common.utils
 
 @JvmInline
-value class InformationSize(private val sizeInBytes: Long) : Comparable<InformationSize> {
+value class InformationSize(val sizeInBytes: Long) : Comparable<InformationSize> {
```

Comment:
The idea behind this class is to mirror `Duration` functionality. So `sizeInBytes` should not be directly accessible but rather converted using e.g. `inWholeBytes` method (which is not present yet).
You can find more developed version of `InformationSize` [here](https://github.com/novasamatech/personhood-android/blob/main/common/src/main/java/com/polkadot/web3citizenship/common/utils/InformationSize.kt)